app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,181 @@
+@import url(http://fonts.googleapis.com/css?family=Titillium+Web&subset=latin,latin-ext);
+@media (min-width: 768px) {
+    .kpx_row-sm-offset-3 div:first-child[class*="col-"] {
+        margin-left: 25%;
+    }
+}
+body {
+    font-family: 'Titillium Web', sans-serif;
+}
+a{
+    color:#ff5400;
+}
+a:hover {
+    opacity: 0.8;
+    color:#ff5400;
+    text-decoration:none;
+}
+.kpx_login .kpx_authTitle {
+    text-align: center;
+    line-height: 300%;
+}
+
+.kpx_login .kpx_socialButtons a {
+    color: white; // In yourUse @body-bg
+	opacity:0.9;
+}
+.kpx_login .kpx_socialButtons a:hover {
+    color: white;
+	opacity:1;
+}
+
+.kpx_login .kpx_socialButtons .kpx_btn-facebook {background: #3b5998; -webkit-transition: all 0.5s ease-in-out;
```

viết css này k ổn lắm
e viết theo kiểu 
```suggestion
.kpx_login  {
      .kpx_socialButtons
             { .kpx_btn-facebook {background: #3b5998; -webkit-transition: all 0.5s ease-in-out;
```

---

tìm hiểu quy tắc xếp chồng trong đây nhé
https://viblo.asia/p/sassscss-la-gi-tim-hieu-cach-viet-css-bang-sassscss-GrLZDxrVZk0

---

em coppy css từ template dài quá nên chưa convert đc ạ